website/docs/users/sample-data/dev-db-setup.md

```diff
@@ -0,0 +1,49 @@
+# Setup Development OMOP Database
+
+## Download the DDL for the OMOP CDM
+Go to https://github.com/OHDSI/CommonDataModel/releases and select the version of the OMOP CDM that applies to you. We use OMOP CDM v5.3.2. Under "Assets", select the `OMOPCDM_*.zip` file, where `*` is the version number.
```

Comment:
I think much of this can be simplified by referring people directly to the OMOP repository readme / docs instead. We don't want to duplicate that and risk becoming out of date.

I would however focus on the use of the SQL DDL scripts they provide, rather than a scripted solution requiring its own runtime dependencies (e.g. the R scripts)

Reply:
Also compare / merge / assimilate / replace the old sample data page.